### DISCUSSION TITLE
Possibility of using jeelizFaceFilter library on top of an HTML <video> tag instead of <canvas> tag?

### DISCUSSION THREAD

**Discussion by Tsajan** [jeeliz/jeelizFaceFilter#246] [Q&A] [answered]
I am using a video conferencing system which is using an HTML `<video>` tag to propagate video feeds to remote participants and I would like to use the `jeelizFaceFilter `to draw something over the participants face. My existing code is using a `<video>` element to display and propagate participants video. In all the documentation provided for `jeelizFaceFilter`, all examples using the canvas tag and then use the `jeelizFaceFilter `functions on top of canvas. My question is if it is possible to use the same functions without having to create a new canvas element.

Thanks!

**Comment by xavierjs** [comment, marked as answer]
You can generate the mediastream from the canvas.
This demo (from a similar library) is doing what you want: https://github.com/jeeliz/jeelizWeboji/tree/master/demos/mediaStream
